src/util/server.c

```diff
@@ -256,6 +256,11 @@ static void default_quit(struct tevent_context *ev,
                          void *siginfo,
                          void *private_data)
 {
+    struct main_context *ctx = talloc_get_type(private_data, struct main_context);
+    if (ctx) {
```

Comment:
This is not necessary. `talloc_free(NULL)` is a noop.

Reply:
fixed